packages/react-instantsearch-dom/src/widgets/Answers.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { createClassNames } from '../core/utils';
+import useAnswers from '../hooks/useAnswers';
+
+const cx = createClassNames('Answers');
+
+/* eslint-disable react/prop-types */
+function DefaultAnswersComponent({ isLoading, hits = [] }) {
+  return (
+    <div className={cx('', hits.length === 0 && '-empty')}>
+      <div className={cx('header')}></div>
+      {isLoading ? (
+        <div className={cx('loader')}></div>
+      ) : (
+        <ul className={cx('list')}>
+          {hits.map((hit, index) => (
+            <li key={index} className={cx('item')}>
+              {JSON.stringify(hit)}
+            </li>
+          ))}
+        </ul>
+      )}
+    </div>
+  );
+}
+
+export default function Answers({
+  searchClient,
```

Comment:
Is the Answers search client always the same as InstantSearch's index? If not, would it make sense to get it from the parent `Index`?

Reply:
It makes sense, and I will address it in the next PR, but how would you access it?

Reply:
Probably by storing the search client in the context?

Reply:
Ok, let's do it in a separate PR